### PR TITLE
DDP-4592: Registration page

### PR DESCRIPTION
**Pull request by fat23cat**
- Added "Are you 18 years old or older?" question
- Texts moved into the translation file
- Improved styles and mobile version
- Added validation messages under fields
- Added loading spinner

### CODE REVIEW

File: ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/invitationServiceAgent.service.ts [hunk 1]
```diff
@@ -44,6 +46,17 @@ export class InvitationServiceAgent extends NotAuthenticatedServiceAgent<any> {
...
-        return new DdpError(serverError.error ? serverError.error.message : '', serverError.error ? serverError.error.code : null);
+        return new DdpError(this.buildErrorMessage(serverError), this.buildErrorType(serverError));
+    }
+
+    private buildErrorMessage(serverError: HttpErrorResponse): string {
+        return serverError.error ? serverError.error.message : '';
+    }
+
+    private buildErrorType(serverError: HttpErrorResponse): ErrorType {
+        if (serverError.error && Object.values(ErrorType).includes(serverError.error.code)) {
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/invitationServiceAgent.service.ts:57]: Could you double-check this line? Under impression that enums don't exist in JavaScript so have not idea what using an enum for argument to Object.values will do. Maybe  switching to checking values of serverError.error.code would be safer?

Comment (fat23cat) [reply ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/invitationServiceAgent.service.ts:57]: How about transforming enum into an object?

Comment (fat23cat) [reply ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/invitationServiceAgent.service.ts:57]: Ah, it won't work as type :(

Comment (fat23cat) [reply ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/invitationServiceAgent.service.ts:57]: I think everything should be okay, `Object.values(ErrorType)` transfrorms into:
image

Comment (MocanaAtBroad) [reply ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/invitationServiceAgent.service.ts:57]: I did same experiment. Guess TypeScript has made some progress!

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**